ElectronicObserver/Window/Tools/EquipmentUpgradePlanner/EquipmentUpgradePlanCostItemModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace ElectronicObserver.Window.Tools.EquipmentUpgradePlanner;
+
+public class EquipmentUpgradePlanCostItemModel
```

Comment:
This could probably be a record instead of a class. You can then delete the `Equals` and `GetHashCode` functions, since records already include them.